src/index.js

```diff
@@ -131,6 +139,14 @@ function addTask() {
   updateIndexes();
 }
 
-renderTasks();
+function clearCompletedTasks() {
+  const completedTasks = savedTasks.filter((task) => task.completed);
+  completedTasks.forEach((task) => {
+    const index = savedTasks.indexOf(task);
+    removeTask(index);
+  });
```

Comment:
- [ ] Also here, Your clear all completed button is expected to clear all the completed tasks from the local storage, In your case, it just unchecks the check box and this is not the expected behavior. Please make sure all the completed tasks are removed totally from the local storage on the button clicked.
- [ ] Also, there is a spelling error in your `clear all completed button`. Please rename `Clear as Complete` to `Clear all Completed`